lib/ansible/modules/infrastructure/stacki/stacki_host.py

```diff
@@ -0,0 +1,324 @@
+                          headers=header)
+
+    rc = stack_r.status_code
+
+    if rc == 200:
+        rc = 0
+
+    return rc, stack_r.text
+
+
+def stack_force_install(result, params, header, client):
+
+    hostname = params['name']
+    endpoint = params['stacki_endpoint']
+    data = dict()
+    rc = None
+    changed = False
+
+    data['cmd'] = "set host boot {} action=install" \
+        .format(hostname)
+    stack_r = client.post(endpoint, data=json.dumps(data), headers=header)
+    rc = stack_r.status_code if stack_r.status_code != 200 else stack_r.status_code
+    out = stack_r.text
+    changed = True
+
+    rc, out = stack_sync(endpoint, header, client)
+
+    result['changed'] = changed
+    result['stdout'] = "api call successful".rstrip("\r\n")
+    result['stderr'] = out.rstrip("\r\n")
+    result['rc'] = rc
+
+
+def stack_add(result, params, header, client):
+
+    hostname        = params['name']
+    rack            = params['rack']
+    rank            = params['rank']
+    appliance       = params['appliance']
+    prim_intf       = params['prim_intf']
+    prim_intf_ip    = params['prim_intf_ip']
+    network         = params['network']
+    prim_intf_mac   = params['prim_intf_mac']
+    endpoint        = params['stacki_endpoint']
+    data            = dict()
+    rc              = None
+    changed         = False
+
+    data['cmd'] = "add host {} rack={} rank={} appliance={}"\
+        .format(hostname, rack, rank, appliance)
+    stack_r = client.post(endpoint, data=json.dumps(data), headers=header)
+    rc = stack_r.status_code
+    out = stack_r.text
+
+    data['cmd'] = "add host interface {} interface={} ip={} network={} mac={} default=true"\
+        .format(hostname, prim_intf, prim_intf_ip, network, prim_intf_mac)
+    stack_r = client.post(endpoint, data=json.dumps(data), headers=header)
+    rc = stack_r.status_code if stack_r.status_code != 200 else stack_r.status_code
+    out = stack_r.text
+    changed = True
+
+    rc, out = stack_sync(endpoint, header, client)
+
+    result['changed'] = changed
+    result['stdout'] = "api call successful".rstrip("\r\n")
+    result['stderr'] = out.rstrip("\r\n")
+    result['rc'] = rc
+
+
+def stack_remove(result, params, header, client):
+
+    hostname        = params['name']
+    endpoint        = params['stacki_endpoint']
+    data            = dict()
+    rc              = None
+
+    data['cmd'] = "remove host {}"\
+        .format(hostname)
+    stack_r = client.post(endpoint, data=json.dumps(data), headers=header)
+    rc = stack_r.status_code
+    out = stack_r.text
+
+    rc, out = stack_sync(endpoint, header, client)
+
+    result['changed'] = True
+    result['stdout'] = "api call successful".rstrip("\r\n")
+    result['stderr'] = out.rstrip("\r\n")
+    result['rc'] = rc
+
+
+def main():
+
+    module = AnsibleModule(
+        argument_spec = dict(
+            state=dict(type='str',
```

Comment:
To be consistent with other modules each of the options should be on a single line (unless they have many choices)